docs/index.rst

```diff
@@ -292,6 +301,58 @@ Misc Options
     a colored output is provided by default when the ``termcolor`` package is
     installed.  This can be disabled by specifying this option.
 
+Tokens configuration
+````````````````````
+
+Tokens supported by Testimony can be configured with ``--tokens``,
+``--minimum-tokens`` and ``--config`` options.
+
+``--tokens`` takes comma-separated list of supported tokens. When testimony
+encounters token outside of this range, it will report it as error.
+
+``--minimum-token`` takes comma-separated list of required tokens. When
+testimony encounters test without all of tokens in this group, it will
+report it as error. Tokens specified here are automatically added to
+list of supported tokens (there is no need to specify single token
+in both ``--minimum-tokens`` and ``--tokens``).
+
+``--config`` is path to YAML configuration file. YAML file should contain
+single map (equivalent of Python dict), where keys are names of tokens
+and values are maps consisting of ``mandatory``, ``type`` and other,
+type-dependant keys. Sample config files are provided in ``tests``
+directory, as well as printed below:
+
+.. code-block:: console
+
+    ---
+    Assert:
+        mandatory: True  # 'Assert' is required in each test
+    Feature:
+        mandatory: True
+    Test:
+        mandatory: True
+    # You can specify that token is not required explicitly, or leave it
+    # out - testimony will assume default value of 'False'
+    # Both tokens below are allowe, but not required
+    BZ:
+        mandatory: False
+    Setup: ""
```

Comment:
Didn't know YAML can do this :) . Changed.

However, I think we still need to validate input value in `config.py:TokenConfig:__init__`, just in case someone prepares not-exactly-correct config file. Right now it invokes `get()` method on value passed in and falls back to empty dict if there is exception.